jwql/website/apps/jwql/views.py

```diff
@@ -402,16 +405,77 @@ def dashboard(request):
         Outgoing response sent to the webpage
     """
 
+    from bokeh.layouts import column, layout
+    from bokeh.embed import components
```

Comment:
Could these imports be put at the top of the module instead of inside this function?